build/terraform/e2e/module.tf

```diff
@@ -41,6 +41,24 @@ variable "kubernetes_versions" {
     "1.27" = ["us-east1", "RAPID"]
     "1.28" = ["us-west1", "RAPID"]
     "1.29" = ["europe-west1", "RAPID"]
+    // "1.30" = ["asia-east1", "RAPID"]
+    //
+    // Before merge: When adding Kubernetes version 1.{N}, first uncomment the line above, extending
+    // the infrastructure to 4 versions temporarily. Come back to these instructions after the
+    // update PR merges.
+    //
+    // After merge: After the Kubernetes update PR merges, and all active PRs are updated:
+    //
+    // * Move the 1.{N-3} line to the bottom and comment it out
+    // * Change the (commented out) 1.{N-3} to 1.{N+1}
+    // * You should now have 3 versions uncommented (versions 1.{N-2} .. 1.{N}),
+    //   and 1.{N+1} commented out for the next update. The new, commented out 1.{N+1}
+    //   should be using the region of the previous 1.{N-3} - this region will become
+    //   unused.
+    //
+    // Rationale: We cycle the regions us-east1 -> us-west1 -> europe-west1 -> asia-east1 -> us-east1
```

Comment:
Are we not considering the `us-central1` region in this cycle?

Reply:
We talked offline, and I’d prefer to keep us-central1 for performance alone so we know they’re not interfering.